src/ai/backend/manager/sweeper/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .kernel import stale_kernel_collection_ctx  # noqa: F401
+from .session import stale_session_collection_ctx  # noqa: F401
```

Comment:
Do you really need to pass it to __init__ one more time?

Reply:
Resolved by: c371dd00f149c9056e7dbf98305e876a967cd63a